src/App.js

```diff
@@ -1,12 +1,188 @@
 import React from 'react';
 import './App.css';
+import Counter from "./Component/Counter";
+import Button from "./Component/Button";
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    //Hier zet ik eerst mijn variabelen op
+
+
+    const [aardbeien, setAardbeien] = React.useState(0);
+    const [bananen, setBananen] = React.useState(0);
+    const [appels, setAppels] = React.useState(0);
+    const [kiwis, setKiwis] = React.useState(0);
+
+    const [voornaam, setVoornaam] = React.useState('');
+    const [achternaam, setAchternaam] = React.useState('');
+    const [leeftijd, setLeeftijd] = React.useState(0);
+    const [postcode, setPostcode] = React.useState('');
+    const [bezorging, togglebezorging] = React.useState('week');
+    const [BezorgTijdslot, toggleBezorgTiidslot] = useState('daytime');
+    const [opmerking, setOpmerking] = React.useState('Plaats hier je opmerkingen');
+    const [termenAkkoord, toggletermenAkkoord] = React.useState(false);
+
+// hier zet ik mijn functies neer.
+    function resetFruit() {
+        setAardbeien(0);
+        setBananen(0);
+        setAppels(0);
+        setKiwis(0);
+    }
+
+    function handleClick() {
+        console.log(`
+    Voornaam: ${voornaam}, 
+    Achternaam: ${achternaam}, 
+    Leeftijd: ${leeftijd}, 
+    Postcode: ${postcode}, 
+    Bezorgfrequentie: ${bezorging},
+    Opmerkingen: ${opmerking},
+    Algemene voorwaarden: ${termenAkkoord}
+    `);
+        console.log(`Fruitmand bestelling - aardbeiden: ${aardbeien}, bananen: ${bananen}, appels: ${appels}, kiwi's: ${kiwis}`);
+
+
+//Hier wordt de app gestart
+        return (
+            <>
+                <h1>Fruitmand bezorgservice</h1>
+
+
+                <article className="counter">
+                    <h2> Bananen</h2>
+                    <Counter
+                        fruitTeller={bananen}
+                        setFruitteller={setBananen}>
+                    </Counter>
+                </article>
```

Comment:
Ik zou `article` en `h2` óók opnemen in het Counter component. Dit zie ik namelijk voor elk fruitsoort terugkomen